src/main/java/com/linkedin/kmf/services/configs/TopicManagementServiceConfig.java

```diff
@@ -44,9 +44,9 @@
       + "replication factor used.";
 
   public static final String TOPIC_CREATION_ENABLED_CONFIG = "topic-management.topicCreationEnabled";
-  public static final String TOPIC_CREATION_ENABLED_DOC = "When true this automatically creates the topic mentioned by \"" +
-      CommonServiceConfig.TOPIC_CONFIG + "\" with replication factor \"" + TOPIC_REPLICATION_FACTOR_CONFIG + "and min ISR of max(" +
-      TOPIC_REPLICATION_FACTOR_CONFIG + "-1, 1) with number of brokers * \"" + PARTITIONS_TO_BROKER_RATIO_CONFIG + "\" partitions.";
+  public static final String TOPIC_CREATION_ENABLED_DOC = "When true this automatically creates the topic mentioned by \""
```

Comment:
The existing style seems better to me. But having `+` at the end of previous line it is more obvious to reader that something else will follow in the next line. If there is no obvious reason to change code style, we usually keep the existing style to avoid back-and-forth changes.